randovania/games/prime2/logic_database/Temple Grounds.txt

```diff
@@ -1496,7 +1496,16 @@ Extra - in_dark_aether: False
   > Pickup (Power Bomb)
       Morph Ball and Power Bomb
   > Event - Dynamo Chamber Gates
-      Scan Visor and Before Dynamo Chamber Gates
+      All of the following:
+          Scan Visor
+          Any of the following:
+              Space Jump Boots or Before Dynamo Chamber Gates
```

Comment:
Why check for before the event on the event connection?

Reply:
probably because if you've done the event, you then can't just walk on through

Reply:
Its possible I misread this